src/Conjure/UI/Model.hs

```diff
@@ -1894,20 +2004,20 @@ rule_TrueIsNoOp = "true-is-noop" `namedRule` theRule where
 
 rule_FlattenOf1D :: Rule
```

Comment:
Do we still need this function?

Reply:
I think that can probably go yes! Should be handled by flattenLex

Reply:
In fact I take it back. We should keep it